docs/pages/docs/divider.mdx

```diff
@@ -40,12 +40,12 @@ Visually or semantically separates content.
                   'Blog',
                   style: typography.sm.copyWith(color: colorScheme.foreground),
                 ),
-                const FSeparator(vertical: true),
+                const FDivider(vertical: true),
                 Text(
                   'Docs',
                   style: typography.sm.copyWith(color: colorScheme.foreground),
                 ),
-                const FSeparator(vertical: true),
+                const FDivider(vertical: true),
                 Text(
                   'Source',
                   style: typography.sm.copyWith(color: colorScheme.foreground),
```

Comment:
`Usage` isn't updated. Might want to do a search on `Separator` to see if anything shows up.